litellm/proxy/utils.py

```diff
@@ -992,7 +993,7 @@ async def check_view_exists(self):
                 return
             else:
                 ## check if required view exists ##
-                if required_view not in ret[0]["view_names"]:
+                if ret[0]["view_names"] and required_view not in ret[0]["view_names"]:
```

Comment:
@krrishdholakia user got this stack trace on discord - could you add a check to see if `ret is not None` ? 


```
2:16:15 - LiteLLM Proxy:DEBUG: utils.py:79 - Backing off... this was attempt #1
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/backoff/_async.py", line 151, in retry
    ret = await target(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/litellm/proxy/utils.py", line 995, in check_view_exists
    if required_view not in ret[0]["view_names"]:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable
```

Reply:
i got the same error. the none value is view_names